agent/command/git_test.go

```diff
@@ -396,7 +396,7 @@ func (s *GitGetProjectSuite) TestStdErrLogged() {
 		if strings.Contains(line.Data, "/invalidRepo.git/' not found") {
 			foundCloneErr = true
 		}
-		if strings.Contains(line.Data, "Permission denied (publickey)") || strings.Contains(line.Data, "Host key verification failed.") {
+		if strings.Contains(line.Data, "Authentication failed for") && strings.Contains(line.Data, "/evergreen-ci/invalidRepo.git") {
```

Comment:
No longer using oauth token so error is slightly different,

Reply:
This test is still erroring without an oauth token so lets remove this for now and I'll fix the test as part of DEVPROD-8701.

Reply:
the change looks good! thank you!